packages/widgets/src/commandpalette.ts

```diff
@@ -652,6 +652,11 @@ namespace CommandPalette {
      */
     readonly isToggled: boolean;
 
+    /**
+     * Whether the command item is toggleable.
+     */
+    readonly isToggleable: boolean;
```

Comment:
Is this bit necessary? Aren't items always toggleable?

Reply:
No, items in the command palette aren't always toggleable. For example, there is a command to open the console.